sormas-backend/src/main/java/de/symeda/sormas/backend/hospitalization/Hospitalization.java

```diff
@@ -178,4 +181,21 @@ public Date getIntensiveCareUnitEnd() {
 	public void setIntensiveCareUnitEnd(Date intensiveCareUnitEnd) {
 		this.intensiveCareUnitEnd = intensiveCareUnitEnd;
 	}
+
+	@Enumerated(EnumType.STRING)
+	public HospitalizationReasonType getReasonForHospitalization() {
+		return reasonForHospitalization;
+	}
+
+	public void setReasonForHospitalization(HospitalizationReasonType reasonForHospitalization) {
+		this.reasonForHospitalization = reasonForHospitalization;
+	}
+
+	public String getOtherReasonForHospitalization() {
```

Comment:
I don't actually know whether annotations for String fields are redundant. In other places, we still use `@Column(columnDefinition = "text")` - have you left it out on purpose because you know it's redundant?

Reply:
Yes, exactly. If the column name is equal to the property name there is no need to add that annotation. But I can add this to be more consistent with the other entities.

Reply:
No, if String maps to "text" automatically, that's perfectly fine. Minimalism is good. :D